server/routes/index.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable no-console */
```

Comment:
@gadishimwe , do not disable any Eslint flag. the purpose of using the Airbnb configuration is to ensure that we write our code in line with a predefined standard. kindly remove all Eslint disabling comments and adjust the codes accordingly